src/pages/Pool/v2.tsx

```diff
@@ -54,6 +54,7 @@ const ButtonRow = styled(RowFixed)`
 
 const ResponsiveButtonPrimary = styled(ButtonPrimary)`
   width: fit-content;
+  border-radius: 12px;
```

Comment:
What about

```
const ResponsiveButtonPrimary = styled(ButtonPrimary).attrs(() => ({ borderRadius: '12px' }))BACKQUOTE
  ${({ theme }) => ...
BACKQUOTE
```

Reply:
In that case, the error was not resolved.

Reply:
Okay, will close.